From fa76242d5606d85c73995f2908382bac30b649e3 Mon Sep 17 00:00:00 2001
From: fli <fupan.li@windriver.com>
Date: Fri, 17 Jun 2016 05:27:05 -0700
Subject: [PATCH] python-smartpm: fix the bug of ssl certificate verify failed

Python from 2.7.9 request explicit Certificate verification
for ssl protocal. smartpm didn't support it,
thus disable Certificate verification by default.

Signed-off-by: fli <fupan.li@windriver.com>
---
 smart/fetcher.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/smart/fetcher.py b/smart/fetcher.py
index 0cac71a..6537546 100644
--- a/smart/fetcher.py
+++ b/smart/fetcher.py
@@ -37,6 +37,9 @@ import re
 import signal
 import threading
 import base64
+if sys.version_info >= (2, 7, 9):
+    import ssl
+    ssl._create_default_https_context = ssl._create_unverified_context
 
 MAXRETRIES = 30
 SPEEDDELAY = 1
-- 
2.7.4

